### ISSUE TITLE
Menu: hover/focus state does not pass 3:1 color contrast requirements (DS6 only)

### ISSUE THREAD

**Issue by seangates** [eBay/skin#633]
Given the scenario where one menu item has `focus` and another is `hover`ed, the user may be confused as to what action will be performed.

![image](https://user-images.githubusercontent.com/105656/58672462-40190480-8304-11e9-864d-da7f8c101e81.png)

The proposal is to use a border with the B6 (#2b0eaf) color to show `focus`:

![image](https://user-images.githubusercontent.com/105656/58672545-abfb6d00-8304-11e9-88be-99d4cd8f74b8.png)

UPDATE (Oct 28th 2019): Changing title and repurposing bug based on a11y issue.

**Comment by ianmcburnie** [comment]
I think we should reinstate the default focus indicator here.

**Comment by seangates** [comment]
Here is what the default looks like:

![image](https://user-images.githubusercontent.com/105656/66346881-5cf7c100-e910-11e9-9675-748023c81214.png)

**Comment by ianmcburnie** [comment]
Hmm, I thought the issue of full focus indicator not appearing had been fixed.

**Comment by seangates** [comment]
That was DS4, because DS4 didn't have a "focus" state.

**Comment by ianmcburnie** [comment]
Yep, I meant the full outline is not appearing.

**Comment by ianmcburnie** [comment]
Also, we should be able to keep the grey background so long as it passes color contrast.

**Comment by seangates** [comment]
Do we feel this is this acceptable?

![image](https://user-images.githubusercontent.com/105656/66353693-dfd44800-e91f-11e9-896c-e5dbe2fb0537.png)

**Comment by ianmcburnie** [comment]
The bottom edge looks a bit funny. Maybe push a draft PR and we can tinker?

**Comment by ianmcburnie** [comment]
I've realised that since we've dropped the bolded focus state, we now definitely need some outline so that we aren't relying on colour alone (i.e. they grey background) to indicate the focussed state.

**Comment by seangates** [comment]
@ianmcburnie I was told by design (after showing them variations and proposals of what to do) to hold steady until 6.5 is delivered for menus. Let's revisit this after 6.5.

**Comment by ianmcburnie** [comment]
Well, the problem as it stands is that we are violating the WCAG "do not use color alone" guideline.

**Comment by seangates** [comment]
"Soon". 😞

**Comment by ianmcburnie** [comment]
Haven't fixed the color contrast, so have mitigated the a11y issue by restoring the focus outline. We can review this again when we get 6.5 handoff from design.
